test/verify/main.cpp

```diff
@@ -74,7 +74,9 @@ int main(int argc, const char* argv[])
                          "test_select_module_add",
                          "test_select_module_reduce",
                          "test_select_module_conv",
-                         "test_split_single_dyn_dim"});
+                         "test_split_single_dyn_dim",
+                         "test_instancenorm_large_3D<migraphx::shape::half_type>",
+                         "test_instancenorm_large_3D<migraphx::shape::float_type>"});
```

Comment:
For some reason CPU backend is failing for  all dtypes on this test i.e. half, float and double.

GPU is also failing on Double. It converts Double to float.